src/hooks/useScanner.js

```diff
@@ -156,6 +160,24 @@ export default function useScanner(enabled, onSuccess) {
         return handleScanRainbowProfile(data);
       }
 
+      if (data.startsWith(COPEPUSH_IOS_PREFFIX)) {
+        if (android) {
+          Alert.alert('Tried to use Android bundle');
+        } else {
+          setDeploymentKey(data.substring(COPEPUSH_IOS_PREFFIX.length));
+        }
+        return;
+      }
+
+      if (data.startsWith(COPEPUSH_ANDROID_PREFFIX)) {
+        if (ios) {
+          Alert.alert('Tried to use iOS bundle');
+        } else {
+          setDeploymentKey(data.substring(COPEPUSH_ANDROID_PREFFIX.length));
+        }
+        return;
+      }
+
```

Comment:
I would move this to fedora handlers as well so we don't spread this code around the app
But it's not a blocker

Reply:
right!